R/pkgs/report.generation/NAMESPACE

```diff
@@ -36,27 +45,39 @@ export(make_scn_time_summary_table_withVent)
 export(make_sparkline_tab_intervention_effect)
 export(make_sparkline_tab_r)
 export(mtr_estimates)
+export(plot_accept_by_location)
+export(plot_accept_by_location_cumul)
+export(plot_accept_by_location_rolling)
 export(plot_county_outcomes)
 export(plot_event_time_by_geoid)
 export(plot_geounit_attack_rate_map)
 export(plot_geounit_map)
 export(plot_hist_incidHosp_state)
+export(plot_hnpi_by_location)
 export(plot_hosp_effec)
+export(plot_hpar_by_location)
 export(plot_inference_r)
 export(plot_line_hospPeak_time_county)
+export(plot_llik_by_location)
+export(plot_llik_total)
 export(plot_model_vs_obs)
 export(plot_needs_relative_to_threshold_heatmap)
 export(plot_outcome_rate)
 export(plot_rt_ts)
 export(plot_scn_outcomes_ratio)
+export(plot_snpi_by_location)
+export(plot_spars)
 export(plot_truth_by_county)
+export(plot_truth_by_location)
 export(plot_ts_hosp_state_sample)
 export(plot_ts_incid_ar_state)
 export(plot_ts_incid_death_state_sample_allPdeath)
 export(plot_ts_incid_inf_state_sample)
+export(pretty_num)
+export(pretty_prop)
 export(print_pretty_date)
 export(print_pretty_date_short)
 export(read_file_of_type)
 export(reference_chunk)
 export(rt_estimates)
-importFrom(magrittr,"%>%")
```

Comment:
I'm concerned that we lost this importFrom. I thought it was important?

Reply:
I think this is a question for @juanderone ... I didn't edit this. maybe this is an accidental carryover from an older branch